pyproject.toml

```diff
@@ -13,7 +13,7 @@ python = ">=3.10,<3.11"
 pydantic = "^1.8.2"
 aerich = "0.5.3"
 asyncpg = "^0.24.0"
-hikari-lightbulb = "^2.1.3"
+hikari-lightbulb = {git = "https://github.com/tandemdude/hikari-lightbulb"}
```

Comment:
Doing this is very risky for a major project. This could mean breaking changes and we'd not even know as we haven't specified the version.

There are 2 options to solve this:
- Use the PyPi version.
- Use specific commit hash while using git link.

Reply:
This is all for testing purposes, 
Lightbulb master branch had something that I wanted to use and was not yet on PyPi so I had to use the GitHub and clone the latest commit.
Will be reverted after new version of Lightbulb is released on PyPi.

Reply:
Okay, for now, you can use the commit hash? Just a precautionary step.

Reply:
Done!